crates/bevy_gizmos/src/aabb.rs

```diff
@@ -0,0 +1,120 @@
+//! A module adding debug visualization of [`Aabb`]s.
+
+use crate as bevy_gizmos;
+
+use bevy_app::{Plugin, PostUpdate};
+use bevy_ecs::{
+    component::Component,
+    entity::Entity,
+    query::Without,
+    reflect::ReflectComponent,
+    schedule::IntoSystemConfigs,
+    system::{Query, Res},
+};
+use bevy_reflect::{std_traits::ReflectDefault, Reflect};
+use bevy_render::{color::Color, primitives::Aabb};
+use bevy_transform::{
+    components::{GlobalTransform, Transform},
+    TransformSystem,
+};
+
+use crate::{
+    config::{GizmoConfigGroup, GizmoConfigStore},
+    gizmos::Gizmos,
+    AppGizmoBuilder,
+};
+
+/// A [`Plugin`] that provides visualization of [`Aabb`]s for debugging.
+pub struct AabbGizmoPlugin;
+
+impl Plugin for AabbGizmoPlugin {
+    fn build(&self, app: &mut bevy_app::App) {
+        app.register_type::<AabbGizmoGroup>()
+            .init_gizmo_group::<AabbGizmoGroup>()
+            .add_systems(
+                PostUpdate,
+                (
+                    draw_aabbs,
+                    draw_all_aabbs.run_if(|config: Res<GizmoConfigStore>| {
+                        config.get::<AabbGizmoGroup>().1.draw_all
+                    }),
+                )
+                    .after(TransformSystem::TransformPropagate),
+            );
+    }
+}
+/// The [`GizmoConfigGroup`] used for debug visualizations of [`Aabb`] components on entities
+#[derive(Clone, Default, Reflect, GizmoConfigGroup)]
+pub struct AabbGizmoGroup {
```

Comment:
I think `AabbGizmoConfigGroup` or `AabbGizmoConfig` would be easier for users to understand.